src/sidebar/components/annotation-action-bar.js

```diff
@@ -1,7 +1,7 @@
 import { createElement } from 'preact';
 import propTypes from 'prop-types';
 
-import uiConstants from '../ui-constants';
+import uiConstants from '../config/ui-constants';
```

Comment:
The contents of `ui-constants.js` isn't related to configuration of the application. It essentially consists of two enums: One for the set of available sidebar panels and one for the set of available tabs. Those could be more usefully represented as union `@typedef`s now (eg. `@typedef PanelName = 'help' | 'loginPrompt' ...`) so they can be referenced in various places.

Reply:
I'm going to open a separate ticket for this small task, as I'd really like to see `ui-constants` go as a module.